eslint.config.js

```diff
@@ -56,11 +55,11 @@ export default [
     },
     ...nodeConfig.map(config => ({
         ...config,
-        files: ["*.(ts|js)", "(gulp|electron)/**/*.(ts|js)"],
+        files: ["*.{ts,js}", "{gulp,electron}/**/*.{ts,js}"],
         ignores: ["gulp/preloader/*.js"],
     })),
     ...runtimeConfig.map(config => ({
         ...config,
-        files: ["src/**/*.(ts|js)x?"],
+        files: ["src/**/*.{ts,js,tsx,jsx}"],
```

Comment:
what about `{ts,js}{,x}` (it does work)

Reply:
it more directly corresponds to the regex and its meaning, but I guess these are all file extensions anyway so enumerating them isn't bad